test/lit/debug/full.wat

```diff
@@ -46,7 +45,6 @@
   ;; FULL-NEXT:    (i32.const 2)
   ;; FULL-NEXT:   )
   ;; FULL-NEXT:  ) ;; end block block
-  ;; FULL-NEXT:  ;;@ src.cpp:1:2
```

Comment:
Was this incorrect before?

Reply:
Unclear what behavior is "correct," but we no longer propagate debug locations to the function epilogue automatically.

Reply:
I would verify that no emscripten debug info tests fail with this change. If that's ok then sgtm.